externs/shaka/text.js

```diff
@@ -246,6 +246,16 @@ shaka.extern.Cue = class {
      */
     this.backgroundColor;
 
+    /**
+     * The number of horizontal and vertical cells into which
+     * the Root Container Region area is divided
+     * cellResolution[0] - columns
+     * cellResolution[1] - rows
+     * @type {!Array.<number>}
+     * @exportDoc
+     */
+    this.cellResolution;
```

Comment:
I feel like this being an array is probably the wrong data type. It will always have exactly two values, so it being an array doesn't have any advantages; it just means that users will have to consult the comments to figure which value is which.
Instead, it should probably be an object with a `columns` and `row` object, or maybe two integers (`cellColumns` and `cellRows`).

Reply:
I guess you are right, `cellResolution` object will provide more details so the code will be more readable, thanks

`cellResolution`  was converted from array to object with `columns` and `rows`